packages/reakit/src/Button/__examples__/ButtonWithTootltip/index.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from "react";
+import { Button } from "reakit/Button";
+import { Tooltip, TooltipReference, useTooltipState } from "reakit/Tooltip";
+import { VisuallyHidden } from "reakit/VisuallyHidden";
+
+type Props = {
+  isLocked: boolean;
+  onClick: () => void;
+};
+
+function LockButton({ isLocked, ...rest }: Props) {
+  const tooltip = useTooltipState();
+  return (
+    <>
+      <TooltipReference {...rest} {...tooltip} as={Button}>
+        {isLocked ? (
+          <>
+            <VisuallyHidden>Click to unlock</VisuallyHidden>{" "}
+            <span aria-hidden>🔒</span>
+          </>
+        ) : (
+          <>
+            <VisuallyHidden>Click to lock</VisuallyHidden>{" "}
+            <span aria-hidden>🔓</span>
+          </>
+        )}
+      </TooltipReference>
+      <Tooltip {...tooltip}>{`It's ${
+        isLocked ? "locked" : "unlocked"
+      }!`}</Tooltip>
```

Comment:
```suggestion
      <Tooltip {...tooltip}>
        It's {isLocked ? "locked" : "unlocked"}!
      </Tooltip>
```
Or abstracting the label into a variable outside of the JSX.

Reply:
I will use your suggestion 👌